src/frontend/src/components/common/Footer/_footer.scss

```diff
@@ -1,28 +1,30 @@
-footer {
+.footer {
   background-color: $primary-colour;
   border-top: 2px solid $accent-colour;
   color: #fff;
   font-family: $font-family;
-  padding-inline: 200px;
 
   /* need to test */
   position: absolute;
   bottom: 0;
   left: 0;
   width: 100%;
-  height: $footer-height;
+  min-height: $footer-height;
+  display: flex;
+  align-items: center;
+  overflow:hidden;
   // top: calc(100vh - 72px);;
```

Comment:
remove comment if not needed - or can you please add comments justifying the presence.